gordo_components/builder/mlflow_utils.py

```diff
@@ -280,16 +289,20 @@ def get_batch_kwargs(metadata: dict) -> dict:
             # Summary stats per metric
             for sk in subkeys:
                 metric_list.append(
-                    Metric(f"{k}_{sk}", scores[k][f"fold-{sk}"], epoch_now(), 0)
+                    Metric(f"{k}-{sk}", scores[k][f"fold-{sk}"], epoch_now(), 0)
                 )
             # Append value for each fold with increasing steps
             metric_list += [
                 Metric(k, scores[k][f"fold-{i+1}"], epoch_now(), i)
                 for i in range(n_folds)
             ]
+    except KeyError:
+        logger.debug(
+            "Key 'metadata.model.cross-validation.scores' not found found in metadata."
```

Comment:
same as above comments, regarding this message and there are _lots_ in here which should not be in `try`